packages/nexus-sdk/src/Resource/index.ts

```diff
@@ -72,11 +72,17 @@ const Resource = (
       orgLabel: string,
       projectLabel: string,
       payload: ResourcePayload,
+      schemaId?: string,
```

Comment:
This should be stuck inside the `ResourcePayload`, then you can destructure it in the function and only pass the remainder to the body

Reply:
SchemaId is part of the path like orgLabel and projectLabel. So I feel it should be parameter just like  orgLabel/projectLabel.